audience_manager/authorized_api/index.yaml

```diff
@@ -0,0 +1,36 @@
+swagger: '2.0'
+info:
+  title: Public Audience Manager API
```

Comment:
Description of authorization is a part of Platform's team task, so will need to wait until they add it to this repo.

I still think that we need to split those endpoints like is now - because new ones are following new routing schema and are using a JSONAPI format, the old ones probably at some point will be refactored, but right now I'm not going to merge those docs.

Reply:
OK, in that case we should name this API same as in [this](https://github.com/PiwikPRO/PPMS-PublicDocs/pull/23/files#diff-d3f6a2815e3d11d42e7f3a6931b75ed0) file: "Authorized HTTP API".

Reply:
Changed